rudder-inventory-ldap/debian/prerm

```diff
@@ -18,7 +18,14 @@ set -e
 
 
 case "$1" in
-    remove|upgrade|deconfigure)
+    remove)
+      # Ensure OpenLDAP is stopped
+      printf "INFO: Stopping rudder-slapd..."
+      invoke-rc.d rudder-slapd forcestop >/dev/null 2>&1
+      echo " Done"
+    ;;
```

Comment:
Note: on debian, dh_installinit will automatically call ```invoke-rc.d <service> stop``` in prerm, but we wan't to call ```forcestop``` so we do it explicitly.